wasm_utils/src/api_serialization/query.rs

```diff
@@ -0,0 +1,12 @@
+use holochain_core_types::hash::HashString;
+
+#[derive(Deserialize, Default, Debug, Serialize)]
+pub struct QueryArgs {
+    pub entry_type_name: String,
+    pub limit: u32,
+}
+
+#[derive(Deserialize, Default, Debug, Serialize)]
+pub struct QueryResult {
+    pub hashes: Vec<HashString>,
```

Comment:
Could we make this `addresses`? please, for consistency?

Reply:
FYI, this is internal and not public facing.

Reply:
well... it get passed through the wasm interface, so its sorta public-y

Reply:
then I guess we should also change the globals from hash to address like:
DNA_HASH -> DNA_ADDRESS

Reply:
yeah... its currently a mix, but Id love to get to address in all the right places. Just merge this as is if you feel its best, i will unblock